core/block_render_svg_vertical.js

```diff
@@ -974,6 +976,55 @@ Blockly.BlockSvg.prototype.renderDraw_ = function(iconWidth, inputRows) {
 };
 
 /**
+ * Give the block an attribute 'data-shapes' that lists its shape[s]
+ * @private
+ */
+Blockly.BlockSvg.prototype.renderClassify_ = function() {
+  var shapes = [];
+  
+  if(this.outputConnection) {
+    if(this.isShadow_) {
+      shapes.push("input");
```

Comment:
"Input" is a pretty overloaded word in scratch-blocks land. We could do "argument", maybe?